File: shared/src/main/java/io/github/zwieback/familyfinance/util/ViewUtils.java
package io.github.zwieback.familyfinance.util;

import android.view.View;
import androidx.core.view.ViewCompat;

public final class ViewUtils {

    public static boolean isLeftToRightLayoutDirection(View view) {
        return ViewCompat.getLayoutDirection(view) == ViewCompat.LAYOUT_DIRECTION_LTR;
    }

    private ViewUtils() {
    }
}

File: shared/src/main/java/io/github/zwieback/familyfinance/util/ViewUtils.kt
package io.github.zwieback.familyfinance.util

import android.view.View
import androidx.core.view.ViewCompat

object ViewUtils {

    fun isLeftToRightLayoutDirection(view: View): Boolean {
        return ViewCompat.getLayoutDirection(view) == ViewCompat.LAYOUT_DIRECTION_LTR
    }
}
